### PR TITLE
app-editors/dav: add 0.9.0, update sources

### PR DESCRIPTION
**Pull request by fkobi**
- bump EAPI
- update SRC_URI & HOMEPAGE
- remove empty src_configure

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: app-editors/dav/dav-0.9.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Minimal console text editor"
+HOMEPAGE="https://github.com/atsb/dav-text"
+SRC_URI="https://github.com/atsb/dav-text/archive/refs/tags/dav-text-${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (thesamesam) [inline app-editors/dav/dav-0.9.0.ebuild:10]: Why rename it at all?

Comment (thesamesam) [reply app-editors/dav/dav-0.9.0.ebuild:10]: Yeah, don't do it. It's nicer if it matches upstream names as long as it's not generic for e.g. archival purposes and so on.

File: app-editors/dav/Manifest [hunk 2]
```diff
@@ -1 +1,2 @@
 DIST dav-0.8.5.tar.gz 22824 BLAKE2B 969458aa6b914f9cd624554690a88f4c2999dcc6e89db38d13d9ba98db7881a34ce1575ccbdf74b3ecca34345f2ec639cfc805a350e31f856f7d086cf23752de SHA512 cdedff875df392e8525399b40d45cdf4e984b1b014f03d9f5cd5be270eebe0003fcf48e9738f6ac2789138973970bb9fe6d0b8babd20c1df61e193f84ee066b3
+DIST dav-0.9.0.tar.gz 23882 BLAKE2B 6845b40bbb74d57438625c2736758c52fbe4cff3e020fd451dee540f06fb893d71e11469f828845c193e9fbbd8a8e0b31dcb9a6fa29383053b61dd9a9d7b0ce4 SHA512 8bb9d111c25e7e41c7d75ee559aeae47494ff68f19f40d2ba3e2ebca23577538b94ba72b58ccf4e64ef92d9e740ee843fa249ef0e4148971b3037259b4f280c5
```
Comment (arkamar) [inline app-editors/dav/Manifest:2]: You have to regenerate manifest, it does not fit the archive name after last push.

**Review by arkamar — CHANGES_REQUESTED** [review]
emerge gives me this warning:
```
 * QA Notice: The ebuild is installing to one or more unexpected directories:
 *
 *   /usr/local
 *
 * Please fix the ebuild to use correct FHS/Gentoo policy paths.
strip: x86_64-pc-linux-gnu-strip --strip-unneeded -N __gentoo_check_ldflags__ -R .comment -R .GCC.command.line -R .note.gnu.gold-version
   /usr/local/bin/dav
```
Additionally, I noticed that the man page is installed with exec bit set. It is because upstream uses default install mask, see https://github.com/atsb/dav-text/blob/dav-text-0.9.0/Makefile#L21. Maybe it would be nice to change that as well.

File: app-editors/dav/files/dav-0.9.0-manpage-no-exec.patch [hunk 3]
```diff
@@ -0,0 +1,25 @@
+From b8750775a1f38eebe5d49ca1dfff9d6527289d00 Mon Sep 17 00:00:00 2001
```
Comment (arkamar) [inline app-editors/dav/files/dav-0.9.0-manpage-no-exec.patch:1]: The patch is no longer needed, since you decided to do it according to my suggestion.

**Review by arkamar — APPROVED** [review]

**Pull request closed without merging**